### PR TITLE
Remaining style fixes

### PR DESCRIPTION
**Pull request by Pr0methean**
UPDATED: This is now the final PR.

### CODE REVIEW

File: src/main/java/net/glowstone/GlowWorld.java [hunk 1]
```diff
@@ -1714,23 +1806,25 @@ public boolean createExplosion(double x, double y, double z, float power, boolea
...
-    public boolean createExplosion(double x, double y, double z, float power, boolean setFire, boolean breakBlocks) {
+    public boolean createExplosion(double x, double y, double z, float power, boolean setFire,
+            boolean breakBlocks) {
         return createExplosion(null, x, y, z, power, setFire, breakBlocks);
     }
 
     /**
      * Create an explosion with a specific entity as the source.
      *
-     * @param source      The entity to treat as the source, or null.
```
Comment (aramperes) [inline src/main/java/net/glowstone/GlowWorld.java]: For me, IntelliJ will align the description like that automatically. Is that against our new checkstyle? I didn't see such changes in the previous PRs.

Comment (Pr0methean) [reply src/main/java/net/glowstone/GlowWorld.java]: No. Reverted; for me, IntelliJ removes that kind of alignment while reformatting the code to wrap long lines (which I do on one file at a time when there are too many to do manually).

Comment (aramperes) [reply src/main/java/net/glowstone/GlowWorld.java]: This change is present in other files in this PR, you might want to check.

Comment (Pr0methean) [reply src/main/java/net/glowstone/GlowWorld.java]: Reverted throughout GlowWorld; could find no instances involving existing Javadocs in other files.

File: src/main/java/net/glowstone/inventory/GlowMetaSkull.java [hunk 2]
```diff
@@ -133,7 +134,7 @@ public OfflinePlayer getOwningPlayer() {
     /**
      * {@inheritDoc}
      *
-     * When this returns false, it may still succeed asynchronously.
+     * <p>When this returns false, it may still succeed asynchronously.</p>
```
Comment (aramperes) [inline src/main/java/net/glowstone/inventory/GlowMetaSkull.java]: Remove the closing tag

**Review by aramperes — DISMISSED** [review]

File: src/main/java/net/glowstone/inventory/GlowMetaSkull.java (hunk 2 above)
Comment (Pr0methean) [reply src/main/java/net/glowstone/inventory/GlowMetaSkull.java]: Done.

**Review by aramperes — APPROVED** [review]

**Pull request merged**